src/Simple.OData.Client.Core/Fluent/IFluentClient.cs

```diff
@@ -71,6 +71,18 @@ public interface IFluentClient<T, FT>
         /// <returns>Self.</returns>
         FT Key(T entryKey);
 
+        /// Sets the specified alternate entry key.
+        /// </summary>
+        /// <param name="entryKey">The entry key.</param>
+        /// <returns>Self.</returns>
+        FT AltKey(object entryKey);
+        /// <summary>
```

Comment:
```suggestion

        /// <summary>
```

Reply:
I noticed that overloads are grouped together without spaces so I kept it like that. Let me know if you still want that change

Reply:
No that's okay if that's the original way.